src/main/java/com/conveyal/datatools/manager/controllers/api/ProjectController.java

```diff
@@ -360,6 +365,7 @@ public static ScheduledFuture scheduleAutoFeedFetch (Project project, int interv
             return DataManager.scheduler.scheduleAtFixedRate(fetchProjectFeedsJob,
                     delayInMinutes, TimeUnit.DAYS.toMinutes(intervalInDays), minutes);
         } catch (Exception e) {
+            LOG.error("Failed to schedule auto feed fetch");
```

Comment:
Should this register an error with bugsnag?

Reply:
Sure why not?